docs/advanced-features/using-mdx.md

```diff
@@ -64,6 +85,14 @@ The following steps outline how to setup `@next/mdx` in your Next.js project:
      - package.json
    ```
 
+## next-mdx-remote
```

Comment:
These two are intentionally not mentioned as they're recommending to `eval` external sources that are transformed into JS.

Reply:
Remove them from this PR.